packages/cloud-cognitive/src/components/ActionSet/ActionSet.js

```diff
@@ -31,6 +31,7 @@ export const ActionSet = React.forwardRef(
       // The component props, in alphabetical order (for consistency).
       actions,
       className,
+      condensed,
```

Comment:
I gave some thought to `condensed` when creating ActionSet. I don't think this property belongs here. It's setting very specific things from the SidePanel design that don't apply more generally. For example, Tearsheet has no concept of condensed, and shouldn't get the height of either $layout-05 OR $layout-04 applied.

You're absolutely right that the height should be being set on the buttons, not on the container -- that's going to go haywire when they stack! But I think the height should be set on the buttons *by the SidePanel*. I'll try and put a suggestion in the code below as to what I mean. See what you think.

PS if we do decide to go ahead with a prop on ActionSet, we'll have to think how to avoid applying unwanted heights for Tearsheet and anything else that might go on to use ActionSet later, and we'll also need to reapply the changes after #592 is merged because that has substantially changed the structure of ActionSet.js and _action-set.scss.

Reply:
Got it, we can definitely set the height of the buttons from the SidePanel. I'll rework this a little bit.